### ISSUE TITLE
Assignment expression does not get error report in Typescript

### ISSUE THREAD

**Issue by ziofat** [edvardchen/eslint-plugin-i18next#52]
```typescript
let text: string;
if (someBool) {
  text = 'hello';
} else {
  text = 'world';
}
```
The above code have no error emitted when lint with this plugin.
